attesters/hydra-s1-simple/index.ts

```diff
@@ -15,8 +14,9 @@ export default class HydraS1SimpleAttester extends HydraS1Attester {
       badge: {
         name: "ZK Badge: Sismo Digger",
         description: "ZK Badge received by early contributors of Sismo",
-        image: "./badges/sismo_digger.svg",
+        image: "hydra-s1-simple/sismo_digger.svg",
```

Comment:
I don't think badge should be "namespaced" by the attester
If i want to add the same badges for different attester i will need to copy it

Reply:
fixed